### DISCUSSION TITLE
Comparison of the *Monkey extensions

### DISCUSSION THREAD

**Discussion by stdedos** [violentmonkey/violentmonkey#2049] [Q&A]
Is there, even a crude comparison, between all of Mokeys available?

I'm taking as little - or as lot - as you are willing to post.

More importantly, though - I'm wondering if you support importing e.g. a Tampermonkey zip export

I found https://github.com/violentmonkey/violentmonkey/discussions/1682, but it talks about v3; Tampermonkey is now v5

**Comment by tophf** [comment]
Tampermonkey's zip can be imported directly.
#1682 is about GreaseMonkey.
